### ISSUE TITLE
Fsh shell: Bug displaying activation

### ISSUE THREAD

**Issue by ioana-blue** [ibm-functions/composer#16]
I had this happen twice, so it seems like a consistent bug. I list activations, click on the second one, I get the result you can see in the print screen. That's a fragment of the logs (actually probably all the logs for that activation). Also, `fsh shell` goes to lala land and never comes back. 

![screen shot 2017-10-11 at 1 14 56 am](https://user-images.githubusercontent.com/17202292/31423273-c40ce32e-ae21-11e7-8b40-616a7c6fa625.png)

**Comment by ioana-blue** [comment]
This print screen may help debug. 

![screen shot 2017-10-11 at 1 21 15 am](https://user-images.githubusercontent.com/17202292/31423397-9a6f46c8-ae22-11e7-986b-082a811a87e3.png)

**Comment by ioana-blue** [comment]
the ibm cloud token above may be a red herring. one pattern that I identified and *might* help debug: the action was a web action that returned something unexpected like `{body: undefined}` due to a bug in calling openwhisk sdk (it looks like a corner case that may be hard to reproduce). I had the same issue twice in a row. from the same `fsh shell` session I was able to visualize other activations just fine.
